Formula/ffmpeg.rb

```diff
@@ -118,8 +118,11 @@ def install
       --disable-indev=jack
     ]
 
-    # Needs corefoundation, coremedia, corevideo
-    args << "--enable-videotoolbox" if OS.mac?
+    if OS.mac?
+      # Needs corefoundation, coremedia, corevideo
+      args << "--enable-videotoolbox"
+      args << "--enable-neon" if Hardware::CPU.arm?
```

Comment:
Does `--enable-neon` work only on ARM macOS? If not, just leave it out of the `if OS.mac?` conditional. (If you're not sure just leave it out anyway. Someone else can fix it if it's broken.)

Reply:
not sure,  just rebase, let me see what CI says